src/components/oragnisms/ElevatorPitch/index.js

```diff
@@ -37,6 +39,7 @@ const ElevatorPitch = () => {
       setDrawPath(pathLength * -1 * scrollPercentage);
       path.current.style.strokeDashoffset = pathLength - drawPath;
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
https://stackoverflow.com/questions/58866796/understanding-the-react-hooks-exhaustive-deps-lint-rule

Reply:
link z rozwiazaniem -> https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies

Dan abaramov komentarz -> https://github.com/facebook/create-react-app/issues/6880#issuecomment-485912528

Reply:
Dzięki, spróbuje dodatkowo przerobić kod odpowiedzialny za animacje przy scrollu używając Intersection Observer API

Reply:
ok